AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt

```diff
@@ -2738,6 +2745,20 @@ open class DeckPicker :
             updateDeckList()
         }
     }
+
+    /**
+     * Show a dialog that explains no sync can occur during migration.
+     */
+    private fun warnNoSyncDuringMigration() {
+        // TODO: Fetch and display real numbers
+        MaterialDialog(this).show {
+            message(text = resources.getString(R.string.sync_impossible_during_migration, 5, 10))
```

Comment:
only one (fake) parameter if you all like my suggestion of just showing files remaining, vs remaining vs total

```suggestion
            message(text = resources.getString(R.string.sync_impossible_during_migration, 5))
```